query-engine/connectors/query-connector/src/write_args.rs

```diff
@@ -148,14 +148,14 @@ impl WriteArgs {
 
     pub fn add_datetimes(&mut self, model: &ModelRef) {
         let now = PrismaValue::DateTime(Utc::now().into());
-        let created_at_field = model.fields().created_at();
+        // let created_at_field = model.fields().created_at();
         let updated_at_field = model.fields().updated_at();
 
-        if let Some(f) = created_at_field {
-            if self.args.get(f.db_name()).is_none() {
-                self.args.insert(f.into(), now.clone().into());
-            }
-        }
+        // if let Some(f) = created_at_field {
+        //     if self.args.get(f.db_name()).is_none() {
+        //         self.args.insert(f.into(), now.clone().into());
+        //     }
+        // }
```

Comment:
Ah yes, good point. So I figured out that, apparently, this is completely unused and our code was practically dead. Since all tests pass I will remove this now. So yea, good to know: `FieldBehavior` was obsolete as only `UpdatedAt` is used so I removed it and use a simple `is_updated_at` flag on fields instead now.